hack/deploy-logging.sh

```diff
@@ -235,24 +239,50 @@ update_images_in_clo_yaml() {
 wait_for_logging_is_running() {
     # we expect a fluentd running on each node
     expectedcollectors=$( oc get nodes | grep -c " Ready " )
+    echo INFO: Expecting $expectedcollectors collector to start
     # we expect $nodeCount elasticsearch pods
+    es_ready="false"
+    fluent_ready="false"
+    kibana_ready="false"
     wait_func() {
-        local actuales=$( oc get pods -l component=elasticsearch 2> /dev/null | grep -c 'elasticsearch.* 2/2 .*Running' )
+        result=0
+        local actuales=$( oc -n ${LOGGING_NS} get pods -l component=elasticsearch 2> /dev/null | grep -c 'elasticsearch.* 2/2 .*Running' )
         if [ $expectedes -ne ${actuales:-0} ] ; then
```

Comment:
`$expectedes` seems unset before usage in this condition.

Reply:
This value is set leading up to the call to this function.  It's not obvious.  I believe this is something to be revisited as we refactor.